apps/web/tailwind.config.js

```diff
@@ -19,15 +18,24 @@ module.exports = {
       'venice-black-drop-shadow': `0px 2px 4px 0px ${VeniceTheme.dropShadow}`,
     },
     extend: {
-      backgroundColor: {
-        'venice-black': VeniceTheme.black,
+      colors: {
+        ...VeniceTheme,
+        'venice-black': {
+          DEFAULT: VeniceTheme.black,
+          300: '#4e4e4e',
+          500: '#2e2e2e',
+        },
       },
-      colors: VeniceTheme,
+      current: 'currentColor',
       fontFamily: {
         sans: ['Montserrat', ...defaultTheme.fontFamily.sans],
         mono: ['Source Code Pro', ...defaultTheme.fontFamily.mono],
       },
-      textColor: VeniceTheme,
```

Comment:
Don't need this because anything passes to `extend.colors` is available for this already.

<img width="790" alt="image" src="https://user-images.githubusercontent.com/3942264/215315020-41bf53eb-c785-477f-b5a4-2c7f1a423bb5.png">

https://tailwindcss.com/docs/customizing-colors#using-custom-colors